core/pkg/core/storage/storage.go

```diff
@@ -0,0 +1,65 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+	"encoding/json"
+	"database/sql"
+	"mango_truth/core/pkg/core/storage/models"
+	"mango_truth/core/pkg/modules"
+	"github.com/google/uuid"
+	_ "github.com/lib/pq"
+	"github.com/volatiletech/null/v8"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+)
+
+type Storage struct {
+	db boil.ContextExecutor
+}
+
+func NewStorage() *Storage {
+	db, err := sql.Open("postgres", "dbname=mango_truth host=localhost user=zahar password=password sslmode=disable")
+	if err != nil {
+		panic(fmt.Sprintf("Can not connect to the database: %s", err.Error()))
+	}
+	return &Storage{db: db}
+}
+
+func (s *Storage) UpdStatus(status modules.DetectionStatus) {
+	data, err := json.Marshal(status.Verdict)
+	if err != nil {
+		panic(fmt.Sprintf("Can not connect verdict to json in DetectionStatus with requestId=%s, error: %s", status.RequestId, err.Error()))
+	}
+	new_status := models.DetectionStatus{
+		RequestID: status.RequestId.String(),
+		Status: status.Status,
+		Data: null.BytesFrom(data),
+	}
+
+	err = new_status.Insert(context.TODO(), s.db, boil.Infer())
```

Comment:
I think, deadlines do more harm than good here, so we can pass background context (and initialise it as Storage.context) all the time

Reply:
Why? context.TODO() does nothing. It is used only as a placeholder